webapp/src/components/properties/link/link.test.tsx

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react'
+import {render} from '@testing-library/react'
+
+import '@testing-library/jest-dom'
+import Link from './link'
+
+describe('components/properties/link', () => {
+    test('returns link properties correctly', () => {
+        const component = (
+            <Link
+                value={'https://github.com/mattermost/focalboard'}
+                onChange={jest.fn()}
+                onSave={jest.fn()}
+                onCancel={jest.fn()}
+                validator={jest.fn(() => {
+                    return true
+                })}
+            />
+        )
+        const {container} = render(component)
+        expect(container).toMatchSnapshot()
```

Comment:
How is this test related to the changes in `link.scss`?

Reply:
You right.
I used an existing css class at the beginning...
I'm going to fix that 😊

Reply:
That's not possible. 😢 
Jest configured with a mock on scss

```
"moduleNameMapper": {
    "\\.(scss|css)$": "<rootDir>/__mocks__/styleMock.js"
},
```
And styleMock.js is empty export

```
module.exports = {};
```

And I don't know how to do....Any idea?

Reply:
I don't think there is an easy way to check css style inside a unit test.

Reply:
Maybe an other issue  to resolve this problem of css in unit tests and do not block the PR for #967?

Reply:
This is definitely not blocking. And even such unit test has value. Just need approval from core contributors (look at the list of reviewers).